apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -226,11 +222,7 @@ export default class TeacherHomepage extends Component {
         )}
         {isEnglish && this.state.donorBannerName && (
```

Comment:
Since this component is already here, assuming this PR won't actually make it visible for anyone yet? Does `this.state.donorBannerName` control who sees the banner?

Reply:
yep donorBannerName will be null if the banner should not be shown